controls/treeview/templates/empty-template.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Empty Template
+page_title: .NET MAUI TreeView Documentation - Empty Template
+description: Learn more about the Empty Template property of the .NET MAUI TreeView control.
+position: 2
+slug: treeview-empty-template
+---
+
+# Empty Template
+
+The TreeView control provides the ability to specify a template when the `ItemsSource` is null or collection is empty.
+
+It exposes the following properties:
+
+* `EmptyTemplate`(`DataTemplate`)&mdash;Defines the content of the view which is shown when in the view has no items.
+* `EmptyStyle`(`Style` with target type `telerik:ItemsEmptyView`)&mdash;Specifies the style applied to the empty view. For more details review the [TreeView Styling]({%slug treeview-style-overview%}#styling-the-empty-template) article.
+
+**Example with Empty Template definition**
```

Comment:
- Use a complete sentence to introduce the example.
- Don't use **bold** as a heading.